commit 6d00416a87e789440c8b0a784ff19d282b89cc55
Author:     Yuhong Wen <yuhongw@nvidia.com>
AuthorDate: Wed Jan 8 22:52:41 2025 -0500
Commit:     GitHub <noreply@github.com>
CommitDate: Thu Jan 9 11:52:41 2025 +0800

    update the client send_aux_request logging (#3136)
    
    * update the client send_aux_request logging.
    
    * Update the logging message.
---
 nvflare/private/fed/client/client_engine.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/nvflare/private/fed/client/client_engine.py b/nvflare/private/fed/client/client_engine.py
index a3417e28..bde557f7 100644
--- a/nvflare/private/fed/client/client_engine.py
+++ b/nvflare/private/fed/client/client_engine.py
@@ -200,11 +200,11 @@ class ClientEngine(ClientEngineInternalSpec, StreamableEngine):
             secure=secure,
         )
 
-        self.logger.info(f"got aux reply: {reply}")
-
         if len(reply) > 0:
+            self.logger.info(f"got aux reply: {reply}")
             return next(iter(reply.values()))
         else:
+            self.logger.debug("no reply from the server aux message response.")
             return Shareable()
 
     def stream_objects(
